### PR TITLE
Update docs regarding Amplitude Data

### PR DESCRIPTION
**Pull request by alex-tpom6oh**

### CODE REVIEW

**Comment by hugihlynsson** [comment]
#### 📝 Changed routes:


| [**/faqs/yes-you-can-faq**](https://docs-h7rx8sa72.avo-dev.app/faqs/yes-you-can-faq) | [**/publishing/integrations**](https://docs-h7rx8sa72.avo-dev.app/publishing/integrations) |
|:-:|:-:|
|<a href="https://docs-h7rx8sa72.avo-dev.app/faqs/yes-you-can-faq">Screenshot of /faqs/yes-you-can-faq</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/URoYQLjo8DhK999uHJwW-POlnLUb0biaZyuc_-YpvnMcCrKd27MEB211YUvompur_TEVWzonGyodGdY4eEhl2w.png">(view full size)</a> |<a href="https://docs-h7rx8sa72.avo-dev.app/publishing/integrations">Screenshot of /publishing/integrations</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/8X6NZLjgqZcs0ytUShowMlP6OHgDiq9e0eSmOzXNp30TAYqQ0qsSmaExnzThODBGvmXpdcEIUTjv1HlSWuSVxg.png">(view full size)</a> |



| [**…lishing/amplitude-data**](https://docs-h7rx8sa72.avo-dev.app/publishing/publishing/amplitude-data) | [**…ng/publishing/overview**](https://docs-h7rx8sa72.avo-dev.app/publishing/publishing/overview) |
|:-:|:-:|
|<a href="https://docs-h7rx8sa72.avo-dev.app/publishing/publishing/amplitude-data">Screenshot of /publishing/publishing/amplitude-data</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/96ZV6S8iFscKgKlPRwxBwuJwQN3Jve6ChC6XxQAOWE6cZ4O7qbvK8KAh91GxAe0_U_4z4zUvNUyBNVWOrArpWA.png">(view full size)</a> |<a href="https://docs-h7rx8sa72.avo-dev.app/publishing/publishing/overview">Screenshot of /publishing/publishing/overview</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/E9Z5cAjk0oz6W_CkITci7oL9-J7Do-f4qvptoYxtDFoO5i7mnq6oM1ijo9M6tY5qbfT-BRhBMrg0ISsOggHWiQ.png">(view full size)</a> |





1 deleted route:
- **/publishing/publishing/amplitude-govern**

Commit 4f4811a8d1a40a1c2d26cb60ec5de2c3f87328a9 (https://docs-h7rx8sa72.avo-dev.app).

File: pages/publishing/integrations.mdx [hunk 1]
```diff
@@ -61,7 +61,7 @@ internal schema management such as production time validation, your SQL tables,
 
 ### Amplitude Govern
 
-[Publish to Amplitude Govern](/publishing/publishing/amplitude-govern) to sync your analytics schema to downstream
+[Publish to Amplitude Govern](/publishing/publishing/amplitude-data) to sync your analytics schema to downstream
```
Comment (logason) [inline pages/publishing/integrations.mdx:64]: Rename Amplitude Govern here as well?

File: pages/publishing/publishing/amplitude-data.mdx [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+
+The integration to [Amplitude Data](https://help.amplitude.com/hc/en-us/categories/17292443498779-Amplitude-Data) allows for your Tracking Plan to be published into Amplitude Data whenever a branch has been merged. You can also manually trigger a publish by clicking the "Publish to Amplitude Data" button on the integration screen or download a CSV file on the Amplitude Data format, which you can manually import into Amplitude Data. The events to be published can be [filtered by Sources, Destinations and Tags](/publishing/publishing/overview#filtering-events-for-publishing).
+
+## Configure the Amplitude Data integration
+
+To enable the direct integration with Amplitude Data, you need to configure it with the following parameters from Amplitude. They can be found from the project settings in Amplitude. From the left sidebar in your Amplitude workspace click: Settings > Projects > [Your Project Name]. The URL should look like this: `https://analytics.amplitude.com/{org-name}/settings/projects/{project-id}/general`.
+
+- **Amplitude API Key**
+- **Amplitude Secret Key**
```
Comment (logason) [inline pages/publishing/publishing/amplitude-data.mdx:19]: Let me confirm with Amplitude if each project still needs to be configured for this to work: https://avosh.slack.com/archives/C01G4LXUDGE/p1689943090014029

Comment (logason) [reply pages/publishing/publishing/amplitude-data.mdx:19]: We should update the docs accordingly: "customers should reach out to their CSM or customer support. We have a note about this in our docs [here](https://www.docs.developers.amplitude.com/analytics/apis/taxonomy-api/)"

**Comment by logason** [comment]
Jumped in with some fixes here @alex-tpom6oh, can you check out my changes?

**Comment by logason** [comment]
> @logason looks good

Sweet, shipping!

**Review by logason — APPROVED** [review]

**Pull request merged**